paket.dependencies

```diff
@@ -1,13 +1,24 @@
 source http://nuget.org/api/v2
-framework: >= net45
+framework: netstandard2.0
+redirects: force
 
-nuget SharpYaml
-nuget FSharp.Core ~> 4.0.0
+nuget YamlDotNet
```

Comment:
@vasily-kirichenko @object Do you any reason why we should not use `YamlDotNet` and should keep `SharpYaml`?

`YamlDotNet.dll` is easier for net standard TP world, because it does not have external dependencies and should not resolve\redirect them...

Reply:
if the tests pass, I’m ok with yamldotnet.